MSBLOC.Web/Program.cs

```diff
@@ -19,7 +22,41 @@ public static void Main(string[] args)
 
         public static IWebHost BuildWebHost(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(new Program().MSBLOCConfigureAppConfiguration)
                 .UseStartup<Startup>()
                 .Build();
+
+        public void MSBLOCConfigureAppConfiguration(WebHostBuilderContext context, IConfigurationBuilder config)
```

Comment:
It seems like this would be better as an extenstion methid to IConfiguration. context isn't used at all BTW.

Reply:
See: https://joonasw.net/view/aspnet-core-2-configuration-changes